include/ismrmrd/xml.h

```diff
@@ -319,7 +320,19 @@ namespace ISMRMRD
   };
 
   struct IsmrmrdHeader
+  : public Entity
```

Comment:
When should I stop by?

Vlad Dreyvitser [C] | Computer Programmer, NIH/NHLBI
Email: vlad.dreyvitser@nih.gov<mailto:vlad.dreyvitser@nih.gov>
Mobile: +1-703-835-3559
9000 Rockville Pike, Bethesda, MD 20814

From: Michael Hansen <notifications@github.com<mailto:notifications@github.com>>
Reply-To: ismrmrd/ismrmrd <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Wednesday, July 20, 2016 at 8:56 AM
To: ismrmrd/ismrmrd <ismrmrd@noreply.github.com<mailto:ismrmrd@noreply.github.com>>
Cc: "Dreyvitser, Vlad (NIH/NHLBI) [C]" <vlad.dreyvitser@nih.gov<mailto:vlad.dreyvitser@nih.gov>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [ismrmrd/ismrmrd] Added virtual getVersion to Entity class. Made IsmrmrdHeader subclass… (#61)


In include/ismrmrd/xml.h<https://github.com/ismrmrd/ismrmrd/pull/61#discussion_r71518628>:

> @@ -319,7 +320,19 @@ namespace ISMRMRD
>    };
>
>    struct IsmrmrdHeader
> +  : public Entity


This is wrong. This ismrmrd header is the deserialized version of the XML header. Not the actual acquisitions. When the XML header is an "entity" on the wire, we need a different class for it or something. We should discuss.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ismrmrd/ismrmrd/pull/61/files/415ea8584c347a8b9246fcdc11ce389084fee76b#r71518628>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AKtiTErWVoBstSx5oF1oGjH9sAuq4oFeks5qXhr2gaJpZM4JQOqn>.